tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -101,7 +101,7 @@ def test_bfloat16(self, attn_backend, torch_compile):
             task.evaluate(llm)
 
     @parametrize_with_ids("torch_compile", [False, True])
-    @parametrize_with_ids("attn_backend", ["TRTLLM", "FLASHINFER"])
+    @_with_ids("attn_backend", ["TRTLLM", "FLASHINFER"])
```

Ruff (0.12.2)

Undefined name `_with_ids`

(F821)